sqlobject/src/test/java/org/jdbi/v3/sqlobject/TestMapBinder.java

```diff
@@ -141,7 +141,7 @@ public Result map(ResultSet r, StatementContext ctx)
         }
     }
 
-    static class Result
+    public static class Result
```

Comment:
If the class is package-private, then the query runs, the result set is mapped to a `Result` object, and then at the very end, trying to return the `Result` throws an `IllegalAccessError`. Bizarre.